lib/src/main/kotlin/com/lemonappdev/konsist/core/container/KoFileImpl.kt

```diff
@@ -43,6 +43,12 @@ internal class KoFileImpl(private val ktFile: KtFile) : KoFile {
         path.removePrefix(rootPathProvider)
     }
 
```

Comment:
Is it possible that there are several "/src/" in the path? If so, is the value after the first or last "/src/" is the source type which we are looking for?